website/content/docs/k8s/helm.mdx

```diff
@@ -63,7 +64,7 @@ Use these links to navigate to a particular top-level stanza.
     the prefix will be `<helm release name>-consul`.
 
   - `domain` ((#v-global-domain)) (`string: consul`) - The domain Consul will answer DNS queries for
-    (Refer to [`-domain`](/consul/docs/agent/config/cli-flags#_domain)) and the domain services synced from
+    (Refer to [`-domain`](https://developer.hashicorp.com/consul/docs/agent/config/cli-flags#_domain)) and the domain services synced from
```

Comment:
is there a reason why you want to use absolute links instead? we prefer relative links whenever possible (same for the rest of these changes)

Reply:
this is auto-generated from the [helm chart](https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml)! feel free to disregard this comment

it makes sense to have absolute links in the helm chart since folks could use that as a source of truth. we might need to update the auto-gen to remove "https://developer.hashicorp.com"

Reply:
@im2nguyen Can you approve as this is blocking the final step of the release. If we want to change the helm chart then that needs to happen in the consul-k8s repo on several branches.

Reply:
i'll pre-approve for now so you're not blocked, but can you revert "Consul Connect" -> "service mesh" it's a change that we would like not be reverted

Reply:
I've reverted the changes for Consul Connect and other places that shouldn't have been changed.